### DISCUSSION TITLE
How can this work on Windows 10 Mobile since it needs 16299?

### DISCUSSION THREAD

**Discussion by benschop-it** [colinkiama/UWP-App-Launcher#117] [Q&A] [answered]
Hi,

Just wondering how this can work on a Windows 10 Mobile device, since you state it needs 16299 and Windows 10 Mobile only supports 15254.603.

Thanks,
Peter

**Comment by IsaacMorris1980** [comment]
Will recheck i thought when i did my research on the sdk's that it would allow it if it was 16299  if not will revert back to previous sdks

**Comment by IsaacMorris1980** [comment]
@benschop-it   i released version 2.0.11.0 that was both requiring 16299  and the person in #61 was able to test on their windows 10 phone and verify it worked so i should work on windows phones

**Reply by benschop-it** [reply]
That surprises me. Which Visual Studio version do you use? I have a 950XL and I will try it out when I have some time. Thanks for your quick response!

**Reply by IsaacMorris1980** [reply]
@benschop-it   was it able to run on your device?

**Comment by IsaacMorris1980** [comment]
running visual studios 2022

**Reply by benschop-it** [reply]
I guess you have not deployed to phone then, I think VS 2017 is the last version that supports UWP apps on mobile devices.

**Reply by IsaacMorris1980** [reply]
VS 2022 should as long as the SDK are installed on the computer  i had VS 2017 and upgraded to VS 2022 as soon as it became available last 3 releases were created in VS 2022

**Comment by IsaacMorris1980** [comment]
last version before moved to 16299 is 1.25 which was actually 1.3.2 i had to fix a certificate issue

**Comment by IsaacMorris1980** [comment, marked as answer]
@benschop-it  newest builds have been reverted back to 15063
